batcher/aligned/src/main.rs

```diff
@@ -464,12 +479,17 @@ async fn main() -> Result<(), AlignedError> {
                 })?;
 
             let keystore_path = &deposit_to_batcher_args.keystore_path;
+            let private_key = &deposit_to_batcher_args.private_key;
 
             let mut wallet = if let Some(keystore_path) = keystore_path {
                 let password = rpassword::prompt_password("Please enter your keystore password:")
                     .map_err(|e| SubmitError::GenericError(e.to_string()))?;
                 Wallet::decrypt_keystore(keystore_path, password)
                     .map_err(|e| SubmitError::GenericError(e.to_string()))?
+            } else if let Some(private_key) = private_key {
+                private_key
+                    .parse::<LocalWallet>()
+                    .map_err(|e| SubmitError::GenericError(e.to_string()))?
             } else {
                 warn!("Missing keystore used for payment.");
```

Comment:
the missing info is not necesarrily a keystore, it can be private key. so text should be like "missing keystore or private key used for payment"